extensions/api/data-management/catalog/src/main/java/org/eclipse/dataspaceconnector/api/datamanagement/catalog/CatalogApiController.java

```diff
@@ -0,0 +1,50 @@
+/*
+ *  Copyright (c) 2022 Bayerische Motoren Werke Aktiengesellschaft (BMW AG)
+ *
+ *  This program and the accompanying materials are made available under the
+ *  terms of the Apache License, Version 2.0 which is available at
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  SPDX-License-Identifier: Apache-2.0
+ *
+ *  Contributors:
+ *       Bayerische Motoren Werke Aktiengesellschaft (BMW AG) - improvements
+ *
+ */
+
+package org.eclipse.dataspaceconnector.api.datamanagement.catalog;
+
+import jakarta.ws.rs.Consumes;
+import jakarta.ws.rs.GET;
+import jakarta.ws.rs.Path;
+import jakarta.ws.rs.Produces;
+import jakarta.ws.rs.QueryParam;
+import jakarta.ws.rs.container.AsyncResponse;
+import jakarta.ws.rs.container.Suspended;
+import jakarta.ws.rs.core.MediaType;
+import org.eclipse.dataspaceconnector.api.datamanagement.catalog.service.CatalogService;
+
+@Path("/catalog")
+@Consumes({ MediaType.APPLICATION_JSON })
+@Produces({ MediaType.APPLICATION_JSON })
```

Comment:
Maybe we should move this also to the method as @paullatzelsperger done it in https://github.com/eclipse-dataspaceconnector/DataSpaceConnector/pull/1141 for the other api endpoints to have everything aligned

Reply:
I did like this to be prepared for this: https://github.com/eclipse-dataspaceconnector/DataSpaceConnector/issues/1175
But, for sure I can remove `@Consumes` as no endpoint consumes json, but the `@Produces` is correct.